<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reflexive Parser</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item row="7" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit_temin"/>
    </item>
    <item row="10" column="0">
     <widget class="QLineEdit" name="lineEdit_metavar"/>
    </item>
    <item row="10" column="9">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Match</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPlainTextEdit" name="plainTextEdit_2_nonterm"/>
    </item>
    <item row="1" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit_gramm"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Grammar:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="9">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Program:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Terminals:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="7">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="9">
     <widget class="QPlainTextEdit" name="plainTextEdit_patt"/>
    </item>
    <item row="7" column="5">
     <widget class="QPlainTextEdit" name="plainTextEdit_regexp"/>
    </item>
    <item row="0" column="9">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Pattern:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>3</width>
        <height>1</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="5">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Regular expressions:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="9">
     <widget class="QPlainTextEdit" name="plainTextEdit_prog"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Metavariable Specifier:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>NonTerminals:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="8">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>3</width>
        <height>1</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1120</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
